starter-code/public/scripts/controllers/aboutController.js

```diff
@@ -7,9 +7,13 @@ var app = app || {};
   aboutController.index = () => {
     $('#about').show().siblings().hide(); // REVIEW: We have a slight refactor in selectors here, which has reduced the amount of code from the last lab.
 
-    // TODO: Call a function to load all the data.
+    // DONE TODO: Call a function to load all the data.
     // Pass a view function as a callback, so the view will render after the data is loaded.
 
+    module.repos.requestRepos(function (repos) {
```

Comment:
This function is not needed. You just need to call module.repoVIew.index as a callback for requestRepos. repoView.index does not take a parameter so there should not be one here.